Makefile

```diff
@@ -63,6 +63,9 @@ develop:
 install: clean
 	$(PIP) install -e ."[api, datadog, prometheus, elasticsearch, pubsub, cloud_monitoring, bigquery, dev]"
 
+uninstall: clean
```

Comment:
I added this target so I could test different versions of the `google-cloud`monitoring` package and make sure I start from a clean slate.